### PR TITLE
Fix README code example and change message id to a string

### PR DESCRIPTION
**Pull request by scd31**
Two changes:

- The code examples in the README/lib.rs have become out of date. I updated them in this PR
- I changed the message id from an integer to a string. There's nothing actually saying it needs to be an int, so this will prevent it from rejecting valid packets.

I probably should have done this in two PRs but I figured the changes were small enough to combine them. I can break this apart if you want. Thanks!

### CODE REVIEW

**Comment by olivia-fl** [comment]
Since this is a breaking change, you may want to bump the version number in this PR. Otherwise looks good to me :)

File: src/packet.rs [hunk 1]
```diff
@@ -149,7 +149,7 @@ mod tests {
             AprsData::Message(msg) => {
                 assert_eq!(msg.addressee, "DEST");
                 assert_eq!(msg.text, "Hello World! This msg has a : colon ");
-                assert_eq!(msg.id, Some(32975));
+                assert_eq!(msg.id, Some("3a2B975".to_string()));
```
Comment (CobaltCause) [inline src/packet.rs]: Minor: instead of `.to_string()` on the RHS, you could use `msg.id.as_deref()` for the LHS

Comment (olivia-fl) [reply src/packet.rs]: I think `assert_eq!(msg.id, Some("3a2B975"))` would work on it's own. [String implements `PartialEq<str>`](https://doc.rust-lang.org/std/string/struct.String.html#impl-PartialEq%3C%26%27a%20str%3E).

**Comment by Turbo87** [comment]
@Benjamin-L I highly recommend against bumping version numbers in PRs. It makes it quite difficult to batch up multiple breaking changes.

File: src/packet.rs (hunk 1 above)
Comment (scd31) [reply src/packet.rs]: Good catch, should be fixed now

**Pull request merged**